dm/loader/lightning.go

```diff
@@ -98,12 +101,15 @@ func makeGlobalConfig(cfg *config.SubTaskConfig) *lcfg.GlobalConfig {
 	lightningCfg.TiDB.Port = cfg.To.Port
 	lightningCfg.TiDB.StatusPort = cfg.TiDB.StatusPort
 	lightningCfg.TiDB.PdAddr = cfg.TiDB.PdAddr
+	lightningCfg.TiDB.LogLevel = cfg.LogLevel
 	lightningCfg.TikvImporter.Backend = cfg.TiDB.Backend
 	lightningCfg.PostRestore.Checksum = lcfg.OpLevelOff
 	if cfg.TiDB.Backend == lcfg.BackendLocal {
 		lightningCfg.TikvImporter.SortedKVDir = cfg.Dir
 	}
 	lightningCfg.Mydumper.SourceDir = cfg.Dir
+	lightningCfg.App.Config.Level = cfg.LogLevel
+	lightningCfg.App.Config.File = cfg.LogFile
```

Comment:
We should net let lightning init log another time. https://github.com/pingcap/tidb/pull/29291/ this PR will skip init lightning log so lightning will share the global config with dm.

Reply:
fixed ptal